docs/extend.md

```diff
@@ -32,3 +32,21 @@ export default function ({ $axios, redirect }) {
   })
 }
 ```
+
+### Create new axios instance based on defaults
+
+If you need to create your own axios instance which based on $axios defaults, you can use `create` method.
+
+```js
+export default function ({ $axios, redirect }) {
+  const newInstance = $axios.create({
+    headers: {
+      common: {
+        Accept: 'text/plain, */*'
+      }
+    }
+  })
+
+  newInstance.setBaseURL('https://yourapi.com')
```

Comment:
Maybe we could document how to inject instance to CTX too?

Reply:
Sure, I can add it